packages/api/src/routes/medical/internal-cq.ts

```diff
@@ -356,6 +358,7 @@ router.post(
   asyncHandler(async (req: Request, res: Response) => {
     // TODO validate the request with the Zod schema, its mostly based on outboundDocumentRetrievalRespSchema
     processOutboundDocumentRetrievalResps(req.body);
+    patientAugmentation(req.body);
```

Comment:
`patientAugmentation` should happen at the end of each HIE pd

Reply:
Done.